templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a class="btn btn-primary" href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a class="btn btn-danger" href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">Delete</a>
+              </td>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.